### DISCUSSION TITLE
How to run more one one test project?

### DISCUSSION THREAD

**Discussion by defunky** [stryker-mutator/stryker-net#2042] [Q&A] [answered]
I'm sure you guys have multiple tests projects, how are you guys executing stryker as it seems like you can only run specific projects.
I have a solution with multiple test projects, and want to run stryker to execute on all test projects it finds

My project looks something like this:
- root
  - project.sln
  - tests
    - Test project1
      - test project.csproj 
    - Test project1
      - test project2.csproj 
  - src
    - project1
      - project1.csproj
    - project2
      - project.csproj

I've tried pointing at sln directly and running but it mentions it finds no csproj files, I assume it looks in the root folder tests (?). I tried having working directly as test/ folder but again I don't think it actually recursively looks up projects and execute them. Is the only way to specify each project one by one? Ideally what I'm looking for is simple pipeline task that executes on all test projects under a given directory

**Comment by rouke-broersma** [comment, marked as answer]
At this time running stryker on a whole solution is not functioning. You can use multiple test projects targeting one project.

**Reply by defunky** [reply]
Not sure I follow, what do you mean by testing multiple projects by targeting one project?

**Reply by rouke-broersma** [reply]
Some teams use unit tests and specflow tests for example. These see often separate test projects. We support testing multiple test projects at the same time as long as they target the same code base.

**Reply by defunky** [reply]
So if I have project1 that has project1.unit.tests and project1.spec.tests; I can run them both in one go? How would you go about doing that. 
I guess that only really solves half the problem. In regards to solution parameter not working, is that something being actively worked on?

**Reply by rouke-broersma** [reply]
You can use the test projects configuration option to set that up.

We are working on solution runs.
